src/test/java/com/conveyal/gtfs/loader/JDBCTableWriterTest.java

```diff
@@ -360,6 +349,288 @@ public void canCreateUpdateAndDeleteRoutes() throws IOException, SQLException, I
         assertThatSqlQueryYieldsZeroRows(getColumnsForId(createdRoute.id, routeTable));
     }
 
+    @Test
+    public void canCreateUpdateAndDeleteBookingRules() throws IOException, SQLException, InvalidNamespaceException {
```

Comment:
These create-update-delete tests look very similar. Is there a way to refactor them?

Reply:
I'm not sure it is, although each test is similar, the values being checked are all different. What did you perhaps have in mind?